packages/grid/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -97,6 +97,24 @@ export const GridRootStyles = styled('div', {
   const borderColor = getBorderColor(theme);
   const radius = theme.shape.borderRadius;
 
+  const columnHeadersStyles = {
+    [`.${gridClasses.columnSeparator}, .${gridClasses['columnSeparator--resizing']}`]: {
+      visibility: 'visible',
+      width: 'auto',
+    },
+  };
+
+  const columnHeaderStyles = {
+    [`& .${gridClasses.iconButtonContainer}`]: {
+      visibility: 'visible',
+      width: 'auto',
+    },
+    [`& .${gridClasses.menuIcon}`]: {
+      width: 'auto',
+      visibility: 'visible',
+    },
+  };
```

Comment:
Move these variables outside the function scope?

Reply:
👍